ament_pep257/ament_pep257/main.py

```diff
@@ -42,19 +42,44 @@ def main(argv=sys.argv[1:]):
     parser = argparse.ArgumentParser(
         description='Check docstrings against the style conventions in PEP 257.',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument(
+    err_code_group = parser.add_mutually_exclusive_group()
+    err_code_group.add_argument(
         '--ignore',
-        nargs='*',
-        default=[
-            'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D106', 'D107',
-            'D203', 'D212', 'D404',
-        ],
-        help='The pep257 categories to ignore')
+        nargs='+',
+        action='extend',
+        default=[],
+        help='Choose the list of error codes for pydocstyle NOT to check for.')
+    err_code_group.add_argument(
+        '--select',
+        nargs='+',
+        action='extend',
+        default=[],
+        help='Choose the basic list of error codes for pydocstyle to check for.'
+    )
+    err_code_group.add_argument(
+        '--convention',
+        choices=pydocstyle.conventions,
+        default='ament',
+        help=f'Choose a preset list of error codes. Valid options are {pydocstyle.conventions}'
```

Comment:
@Arnatious will `ament` show up in `pydocstyle.conventions`? I'd expect `ament_pep257 --convention=ament` not to fail, even if redundant.

Reply:
a4d6399 added logic to populate/allow "ament" as an option